wdl-analysis/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Added utility function `analyze` to simplify the API ([#283](https://github.com/stjude-rust-labs/wdl/pull/283)).
```

Comment:
```suggestion
* Added `analyze` function to simplify the API ([#283](https://github.com/stjude-rust-labs/wdl/pull/283)).
```

Reply:
In general, I find calling things "utility" functions or making "utility" modules to be not the best route. I was in favor of you pulling this out of the `util` module, but I think we should also just not refer to it as a utility function.